scripts/globals/mobskills/Immortal_Anathema.lua

```diff
@@ -0,0 +1,25 @@
+---------------------------------------------
+--  Immortal Anathema
+--
+--  Description: Inflicts a curse on all targets in an area of effect.
+--  Type: Enfeebling
+--  Utsusemi/Blink absorb: Ignores shadows
+--  Range: AoE
+--  Notes: 
+---------------------------------------------
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+
+---------------------------------------------
+function onMobSkillCheck(target,mob,skill)
+    return 0;
+end;
+
+function onMobWeaponSkill(target, mob, skill)
+    local typeEffect = EFFECT_CURSE_I;
+
+    skill:setMsg(MobStatusEffectMove(mob, target, typeEffect, 25, 0, 300));

```

Comment:
change this as well as the other skill:setMsg's to set it like so
```
local msg =  MobStatusEffectMove(mob, target, typeEffect, 25, 0, 300);

skill:setMsg(msg);
```

cause readability